### ISSUE TITLE
Cygwin: Compilation error during make check

### ISSUE THREAD

**Issue by mkoeppe** [flatsurf/e-antic#108]
As reported in https://trac.sagemath.org/ticket/27952#comment:109

on cygwin-standard during make check of e-antic:

```
e_antic-0.1.7] renfxx/test/t-constructor.cpp: In function ‘void check_equal(renf_elem_class&, mp_limb_signed_t)’:
[e_antic-0.1.7] renfxx/test/t-constructor.cpp:29:11: error: ambiguous overload for ‘operator!=’ (operand types are ‘renf_elem_class’ and ‘mp_limb_signed_t’ {aka ‘long long int’})
[e_antic-0.1.7]    29 |     if (a != b)
[e_antic-0.1.7]       |         ~ ^~ ~
[e_antic-0.1.7]       |         |    |
[e_antic-0.1.7]       |         |    mp_limb_signed_t {aka long long int}
[e_antic-0.1.7]       |         renf_elem_class
(with g++ version... 9.3.0)
```

**Comment by mkoeppe** [comment]
... as can be checked by merging #103 ...

**Comment by saraedum** [comment]
I don't think I want to do anything about this in the pre 1.0.0 releases. I am not sure if this is still an issue in 1.0.0. We can probably check once #168 has been resolved.

**Comment by saraedum** [comment]
Is this still an issue in the 1.x releases?

**Comment by mkoeppe** [comment]
See https://trac.sagemath.org/ticket/31588 for the current status

**Comment by saraedum** [comment]
Ok. I commented on everything there. Everything should be fixed. The isnormal error is strange. Presumably there is a fix in #234 which will be part of the next patch release.

**Comment by saraedum** [comment]
So, the original error does not show up in that ticket, unless I missed it somehow.
